src/test/utils.tsx

```diff
@@ -82,3 +88,22 @@ export const hasToast = async (
   expect(card).toBeInTheDocument();
   expect(card).toHaveAttribute("data-type", severity);
 };
+
+export const hasEmptyState = async (
+  title: string,
+  message?: string,
+  buttonLabel?: string,
+) => {
```

Comment:
I think this is probably checking more than it needs to. We don't need to check all the details of the empty state component each time it is rendered. It might be enough to check that the empty state component exists by giving it a test id. The other details could get moved to the NoEntityCard tests.